src/components/detail/ArtistReleases.jsx

```diff
@@ -0,0 +1,30 @@
+// get artist id from searchParams
+// make request to API with artist id to get artist details
+// check if cover art exists
+// if exists, get from URL and display; else display placeholder img
+// render list of releases with name header
+import React, { useState } from 'react';
+import useArtist from '../state/useArtist';
+import ReleaseList from './ReleaseList';
+import { useLocation } from 'react-router-dom';
+
+const ArtistReleases = () => {
+  const [pages, setPages] = useState(1);
+  const { releases } = useArtist(pages);
+  const { state } = useLocation();
+ 
+  return (
+    <>
+      <button 
```

Comment:
lines 18 to 23 could be it's own component (if you had places that had pagination). You could make it a presentational component but pass in the component as needed.